shared/test/test_assets.rb

```diff
@@ -154,6 +154,24 @@ def test_assets_quota
     end
   end
 
+  def test_asset_last_modified
+    channel = create_channel(@channels)
+
+    put @assets, channel, 'test.png', 'version 1', 'image/png'
+    get @assets, channel, 'test.png'
+    v1_last_modified = @assets.last_response.headers['Last-Modified']
+
+    sleep 1
```

Comment:
Can't mock this since we need the timestamps on the remote S3 server to be different.